music21/romanText/translate.py

```diff
@@ -574,6 +575,10 @@ def translateMeasureLineToken(self, measureLineToken: rtObjects.RTMeasure):
 
         else:
             m = self.translateSingleMeasure(measureLineToken)
+            if self.tsCurrent is not None and m.duration.quarterLength == 0:
+                m.duration = duration.Duration(
+                    self.tsCurrent.barDuration.quarterLength
+                )
```

Comment:
this isn't right, unfortunately.  Duration on a stream is special property that automatically manages its quarterLength based on the duration of elements inside of it.  You'd be repeating an empty measure, not a measure of RN.

What should happen, I think is that the code from 593-605 (was 588-600) from "fillToMeasureToken" should be abstracted out into a new method (`fillMeasureFromPreviousRn(mFill: stream.Measure) -> None`) and then that method should be called instead.  Something needs to go into the repeat measure.  We can't repeat an empty span of time.

Reply:
OK, I'll do this.

In this connection, it might be worth clarifying the docs on when (if ever) explicitly setting duration is possible:

https://web.mit.edu/music21/doc/moduleReference/moduleStreamBase.html#music21.stream.base.Stream.duration

They describe explicitly setting duration in the way I did above as an "advanced usage". I assumed an "advanced usage" would be ok inside the music21 source which is why I thought it would be ok to do.

Reply:
Yeah, I'll try to improve the docs for that.

It basically means that no matter how long the duration of the measure is, it'll always report some other number, which is something that an end user might want to do, but which our parsers shouldn't do.